tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -251,120 +252,145 @@ class ProjectUpdateTestTokenAuthenticated(APITestCase):
     def setUp(self):
         super().setUp()
         self.project = self.create_project(platform="javascript")
+        self.team = self.project.teams.first()
         self.user = self.create_user("bar@example.com")
+        self.url = reverse(
+            "sentry-api-0-project-details",
+            kwargs={
+                "organization_slug": self.project.organization.slug,
+                "project_slug": self.project.slug,
+            },
+        )
+        self.data = {"platform": "rust"}
+
+    def create_bearer_token(self, user, scope_list: List[str]) -> str:
+        token = ApiToken.objects.create(user=user, scope_list=scope_list)
+        return f"Bearer {token.token}"
 
-    def test_member_can_read_project_details(self):
+    def test_member_can_update_limited_project_details(self):
         self.create_member(
             user=self.user,
             organization=self.project.organization,
-            teams=[self.project.teams.first()],
+            teams=[self.team],
             role="member",
         )
 
-        token = ApiToken.objects.create(user=self.user, scope_list=["project:read"])
-        authorization = f"Bearer {token.token}"
+        authorization = self.create_bearer_token(self.user, ["project:read"])
 
-        url = reverse(
-            "sentry-api-0-project-details",
-            kwargs={
-                "organization_slug": self.project.organization.slug,
-                "project_slug": self.project.slug,
-            },
+        response = self.client.get(
+            self.url, format="json", HTTP_AUTHORIZATION=authorization, data={"isBookmarked": True}
         )
-        response = self.client.get(url, format="json", HTTP_AUTHORIZATION=authorization)
         assert response.status_code == 200, response.content
```

Comment:
Could we do some assertions on `response.content` rather than just making sure it exists?

Reply:
Added assertions